vispy/scene/canvas.py

```diff
@@ -87,17 +87,20 @@ def __init__(self, *args, **kwargs):
         self._fb_stack = []  # for storing information about framebuffers used
         self._vp_stack = []  # for storing information about viewports used
         self._scene = None
-        self._bgcolor = Color(kwargs.pop('bgcolor', 'black')).rgba
```

Comment:
The splitting over two lines is because the `self.update()` in the setter must be called after the super `__init__`, but the argument must be popped from the kwargs before that happens. I could add a check before calling self.update() instead, I think, would that be better?